AutoClicker/Views/MainWindow.xaml.cs

```diff
@@ -278,6 +319,56 @@ private void UnregisterHotkey(int hotkeyId)
 
         #region Event Handlers
 
+#if DEBUG
+        [DllImport("User32.dll")]
+        public static extern IntPtr GetDC(IntPtr hwnd);
+        [DllImport("User32.dll")]
+        public static extern void ReleaseDC(IntPtr hwnd, IntPtr dc);
+#endif
+        private void GloablMouseHook_MouseAction(object sender, EventArgs e)
+        {
+            Point newMousePosition = GloablMouseHook.GetCursorPosition();
+            
+            // Use the Pythagorean Theorem to calculate the amount of pixels the mouse has moved
+            int moveX = newMousePosition.X - _lastMousePosition.X;
+            int moveY = newMousePosition.Y - _lastMousePosition.Y;
+            int mouseTravel = (int)Math.Sqrt(moveX * moveX + moveY * moveY);
+
+            switch (AutoClickerSettings.ToleranceMode)
+            {
+                case ToleranceMode.Absolute:
+                    break;
+                case ToleranceMode.Relative:
+                    _lastMousePosition = newMousePosition;
+                    break;
+                case ToleranceMode.Intelligent:
+                    _mouseMoveDelta.Stop();
+                    // If the mouse doesn't move for one second this is it's new home. The faster the mouse moves the lower gets the cathing up of the home-point. 
+                    double deltaMouseMoveSeconds = Math.Min(_mouseMoveDelta.ElapsedMilliseconds / (1000 / Constants.MOUSE_HOOK_MIN_EVENT_DELTA_MILIS), 1.0);
+                    _lastMousePosition.X += (int)(moveX * deltaMouseMoveSeconds);
+                    _lastMousePosition.Y += (int)(moveY * deltaMouseMoveSeconds);
+/*                    
+#if DEBUG
+                    //Directly draw to the screen to debug

```

Comment:
what is the purpose of this commented-out code?

Reply:
It was used for debugging. It draws a little rectangle where the code thinks the home point is.

Reply:
If it's no longer needed we can remove it